.github/workflows/close_stale_issues.yaml

```diff
@@ -0,0 +1,26 @@
+name: "Close stale issues"
+on:
+  schedule:
+  - cron: "5 * * * *"
```

Comment:
This runs every hour, at the 5th minute. If we want to have a better feeling for what this does and have the bot do incremental work on the set of issues / PR we can:

- Set it to run once per day
- Give a limit to the number of issues that the bot will act on (50 /  day?)
- Do a sweep over issues and PRs to mark all the ones that we want to preserve

Opinions on this proposal are welcome.